LadderManager.cpp

```diff
@@ -1057,24 +1091,17 @@ void LadderManager::RunLadderManager()
 			}
 			else
 			{
-				// Terran bug. Skip where terran is player 2
-				/*
-				if (NextMatch.Agent2.Race == sc2::Race::Terran)
-				{
-					continue;
-				}
-				*/
 				result = StartGame(NextMatch.Agent1, NextMatch.Agent2, NextMatch.Map);
 			}
-			UploadMime(result, NextMatch);
+			//UploadMime(result, NextMatch);
```

Comment:
Need to un-comment this one.
Having the disconnected/no curl compile option in future will help avoid having to do this hopefully. :)

Reply:
Argh. I always test it with the this off. Forgot to un-comment this.

Reply:
Yeah it's a tad annoying.